src/backend/base/langflow/components/tavily/__init__.py

```
@@ -1,4 +1,9 @@
-from .tavily_extract import TavilyExtractComponent
 from .tavily_search import TavilySearchComponent
+from .tavily_extract import TavilyExtractComponent
+from .tavily_crawl import TavilyCrawlComponent
 
-__all__ = ["TavilyExtractComponent", "TavilySearchComponent"]
+__all__ = [
+    "TavilySearchComponent",
+    "TavilyExtractComponent",
+    "TavilyCrawlComponent",
+]
```

Ruff (0.11.9)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)